examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>this sentence is not clear "<strong>The worker with more batches waits for the gradients from the other worker, but it finished the training run.".</strong></p><p> </p><p>which worker finished the training run? the one with more batches or less batches?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>